deploy/ui/form.json

```diff
@@ -1180,6 +1180,80 @@
+                    },
+                    {
+                      "label": "Client ID",
+                      "type": "text",
+                      "required": false,
+                      "jsonPath": "$.spec.objects.servers[*].kafka.clientID",
+                      "originalJsonPath": "$.spec.objects.servers[*].kafka.clientID",
+                      "description": "An id string to pass to the server when making requests. Not passed by default (the purpose of this property is to provide a logical application name for logging)"
+                    },
+                    {
+                      "label": "Autocreate Topics",
+                      "type": "checkbox",
+                      "checked": true,
+                      "required": false,
+                      "jsonPath": "$.spec.objects.servers[*].kafka.autocreateTopics",
+                      "originalJsonPath": "$.spec.objects.servers[*].kafka.autocreateTopics",
+                      "description": "Allow automatic topic creation.Allow automatic topic creation. Enabled by default"
+                    },
+                    {
+                      "label": "Group Identifier",
+                      "type": "text",
+                      "required": false,
+                      "jsonPath": "$.spec.objects.servers[*].kafka.groupID",
+                      "originalJsonPath": "$.spec.objects.servers[*].kafka.groupID",
+                      "description": "A unique string that identifies the group this consumer belongs to. Default to 'jbpm-consumer'"
+                    },
+                    {
+                      "label": "Number of Acknowledgments ",
+                      "type": "integer",
+                      "required": false,
+                      "jsonPath": "$.spec.objects.servers[*].kafka.acks",
+                      "originalJsonPath": "$.spec.objects.servers[*].kafka.acks",
+                      "description": "The number of acknowledgments the producer requires the leader to have received before considering a request complete. 1 by default, which means the leader will write the record to its local log but will respond without awaiting full acknowledgement from all followers"
+                    },
+                    {
+                      "label": "Max Block in Milliseconds",
+                      "type": "integer",
+                      "required": false,
+                      "jsonPath": "$.spec.objects.servers[*].kafka.maxBlockMs",
+                      "originalJsonPath": "$.spec.objects.servers[*].kafka.maxBlockMs",
+                      "description": "Number of milliseconds that indicates how long publish method will block. Default 2000 (2 seconds)"
+                    },
+                    {
+                      "label": "Topics",
+                      "required": false,
+                      "jsonPath": "$.spec.objects.servers[*].kafka.topics",
+                      "type": "object",
```

Comment:
is this type right now that its a string slice?

@swati-kale what type should be used here for a string array/slice? []string?

Reply:
Console CR form supports array of objects, where objects may have any fields e.g string.

Reply:
if you want the yaml representation as  below for topics ,
topics:
            - xyz
            - abc
then I will suggest changes

Reply:
"fields": [
                        {
                          "label": "Mapping Message=Topic",
                          "type": "text",
                          "required": false,
                          "jsonPath": "$.spec.objects.servers[*].kafka.topics[*]",
                          "originalJsonPath": "$.spec.objects.servers[*].kafka.topics[*]",
                          "description": "Mapping Message=Topic"
                        }